litellm/proxy/_types.py

```diff
@@ -227,6 +227,14 @@ class LiteLLM_JWTAuth(LiteLLMBase):
     user_id_upsert: bool = Field(
         default=False, description="If user doesn't exist, upsert them into the db."
     )
+    user_email_jwt_field: Optional[str] = Field(
+        default="email",
```

Comment:
@Collen-Roller why use a default str instead of leaving it as None?

Reply:
Because most OIDC providers use the email and this would automatically associate it the user that gets created. It's an entry in the DB so wouldn't it make sense to have a default so the user doesn't have to configure it? Manual override option with `user_email_jwt_field`?